frontend/src/components/maps/leaflet/LayerPopup/hooks/useBoundaryLayer.tsx

```diff
@@ -28,9 +28,6 @@ export const useBoundaryLayer = () => {
   let domain = '';
   let parcelURL = '';
   switch (env.env) {
-    case 'test':
```

Comment:
Won't this make `test` use the same as `prod`?  If we want to use the `dev` you need to add the `case 'test':` before the `case 'development':`

Reply:
You're right, good catch!